src/abstractsector.jl

```diff
@@ -5,6 +5,7 @@ using BlockArrays: blocklengths
 using LabelledNumbers: LabelledInteger, label, label_type, labelled, unlabel, unlabel_type
 using GradedUnitRanges:
   GradedUnitRanges, blocklabels, fuse_blocklengths, gradedrange, tensor_product
+using TensorProducts: ⊗, TensorProducts
```

Comment:
```suggestion
using TensorProducts: TensorProducts, ⊗
```
By convention I like to list the module first.